index.html

```diff
@@ -4,18 +4,23 @@
     <title>Home</title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
+    <script src="https://kit.fontawesome.com/804e0be2a4.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="styles.css" />
     <script type="module" src="script.js"></script>
+    
   </head>
   <body>
     <header>
       <section class="logo">
-        <img src="img/Logo.png" alt="Logo" />
+        <img src="img/MagicQuill.png" alt="Logo" />
       </section>
 
-      <p class="title">WizziQuills</p>
+      <section class="text-section">
+        <p class="title">Turn your scribbles into Spells!</p>
+        <p class="shop-now">Shop Sets now! <i class="fa-solid fa-arrow-right"></i> </p>
+      </section>
 
-      <section class="search"></section>
+      <!-- <section class="search"></section> -->
     </header>
 
     <!-- <section class="artboard">

```

Comment:
Uncomment this code as it has a piece of this page that has been completed.

*My bad that it was uncommented out.